ml-proto/test/block.wast

```diff
@@ -200,7 +200,7 @@
 
 ;; TODO(stack): move this elsewhere
 (module (func $type-break-num-vs-void
-  (block (i32.const 66) (br 0))
+  (block (drop (i32.const 66)) (br 0))
```

Comment:
In the AST model, a value must be used or dropped. The 66 constant is not used, therefore it must be dropped.